zebra-test/src/command.rs

```diff
@@ -221,4 +224,13 @@ impl TestOutput {
         .with_section(command)
         .with_section(stdout)
     }
+
+    /// Returns the exit signal on unix platforms, and the exit code otherwise.
+    pub fn exit_status(&self) -> Option<i32> {
+        #[cfg(unix)]
+        return self.output.status.signal();
+
+        #[cfg(not(unix))]
+        return self.output.status.code();
+    }
```

Comment:
I think we should try to isolate the platform specific behaviour to this function rather than mixing up signals and exit codes and requiring ppl to know that the meaning of the value depends on the platform.

```suggestion
    /// Returns the exit signal on unix platforms, and the exit code otherwise.
    pub fn was_killed(&self) -> bool {
        #[cfg(unix)]
        self.output.status.signal() == Some(9)

        #[cfg(not(unix))]
        self.output.status.code() == Some(1)
    }
```

Reply:
Sounds good to me, i made this change at https://github.com/ZcashFoundation/zebra/pull/814/commits/dee397a2a8fdfc7a1225dbce412fac47f57e2818

I was not able to remove the `return`s in the `was_killed` function as suggested, build issues in that form with the macros.